chromium_src/third_party/blink/common/features.cc

```diff
@@ -17,6 +17,7 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kReducedReferrerGranularity, base::FEATURE_ENABLED_BY_DEFAULT},
 
     {kAdInterestGroupAPI, base::FEATURE_DISABLED_BY_DEFAULT},
+    {kCssSelectorFragmentAnchor, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
sort?

Reply:
Fixed in 75ff228aaed2ff61c7f2a182237ad83cca918e80